barrister/runtime.py

```diff
@@ -530,9 +553,9 @@ def __init__(self, transport, validate_request=True, validate_response=True,
         self.validate_resp = validate_response
         self.id_gen = id_gen
 
-    def start(self):
+    def get_iface(self):
```

Comment:
it makes sense that internally this is called iface, but from a "user" perspective, shouldn't this really be the `get_api`?